to fix docs typo

dharada opened the pull request:
<!-- Type of change
Please label this PR with one of the following labels, depending on the scope of your change:
- Bug
- Enhancement
- Breaking change
- Deprecation
- Cleanup
- Docs
-->

## What does this PR do?

To fix type

see - https://www.elastic.co/guide/en/beats/filebeat/current/exported-fields-aws-cloudwatch.html
see - https://github.com/elastic/beats/blob/d6aeef5a5ac51b4c1f114d34342f7fb3a8ae1289/x-pack/filebeat/input/awscloudwatch/processor.go#L64

<!-- Mandatory
Explain here the changes you made on the PR. Please explain the WHAT: patterns used, algorithms implemented, design architecture, message processing, etc.
-->

elasticmachine commented:
Pinging @elastic/obs-docs (Team:Docs)

dharada commented:
hello @elastic/docs I don't know why but I'm assigned by `mergify` automatically. can you take a look at this ?

elasticmachine commented:
## :broken_heart: Build Failed


<!-- BUILD BADGES-->
> _the below badges are clickable and redirect to their specific view in the CI or DOCS_
[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-31476/detail/PR-31476/1//pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-31476/detail/PR-31476/1//tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-31476/detail/PR-31476/1//changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-31476/detail/PR-31476/1//artifacts) [![preview](https://img.shields.io/badge/docs-preview-yellowgreen)](http://beats_31476.docs-preview.app.elstc.co/diff)  [![preview](https://img.shields.io/badge/elastic-observability-blue)](https://ci-stats.elastic.co/app/apm/services/beats-ci/transactions/view?rangeFrom=2022-05-01T03:23:23.935Z&rangeTo=2022-05-01T03:43:23.935Z&transactionName=Beats/beats/PR-{number}&transactionType=job&latencyAggregationType=avg&traceId=f04d0338f185191354934bb01c70a1df&transactionId=55d92d462c1ab2b2)

<!-- BUILD SUMMARY-->
<details><summary>Expand to view the summary</summary>
<p>

#### Build stats




* Start Time: 2022-05-01T03:33:23.935+0000


* Duration: 11 min 38 sec





</p>
</details>

<!-- TEST RESULTS IF ANY-->


<!-- STEPS ERRORS IF ANY -->












### Steps errors [![2](https://img.shields.io/badge/2%20-red)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-31476/detail/PR-31476/1//pipeline)
  <details><summary>Expand to view the steps failures</summary>
  <p>


  
  
  
  
##### `make check`
<ul>
<li>Took 2 min 23 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/PR-31476/runs/1/steps/297/log/?start=0">here</a></li>
<li>Description: <code>make check</code></l1>
</ul>
  
  
  
  
##### `Error signal`
<ul>
<li>Took 0 min 0 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/PR-31476/runs/1/steps/304/log/?start=0">here</a></li>
<li>Description: <code>Error 'hudson.AbortException: script returned exit code 2'</code></l1>
</ul>
  
  </p>
  </details>



## :grey_exclamation: Flaky test report
No test was executed to be analysed.



## :robot: GitHub comments

To re-run your PR in the CI, just comment with:



- `/test` : Re-trigger the build.


- `/package` : Generate the packages and run the E2E tests.


- `/beats-tester` : Run the installation tests with beats-tester.


- `run` `elasticsearch-ci/docs` : Re-trigger the docs validation. (use unformatted text in the comment!)




<!--COMMENT_GENERATED_WITH_ID_comment.id-->

rdner commented:
@dharada please run `make update` in the root of the repository and commit the changes. There is some generated documentation that has to be updated.

I also found `aws-cloudwatch` in these files:

```
filebeat/docs/fields.asciidoc
filebeat/docs/filebeat-options.asciidoc
metricbeat/docs/modules/aws.asciidoc
metricbeat/docs/modules/aws/cloudwatch.asciidoc
metricbeat/docs/modules_list.asciidoc
x-pack/filebeat/_meta/config/filebeat.inputs.reference.xpack.yml.tmpl
x-pack/filebeat/build/package/module/awsfargate/log/config/aws-cloudwatch.yml
x-pack/filebeat/build/package/module/awsfargate/log/manifest.yml
x-pack/filebeat/docs/inputs/input-aws-cloudwatch.asciidoc
x-pack/filebeat/fields.yml
x-pack/filebeat/filebeat.reference.yml
x-pack/filebeat/input/awscloudwatch/_meta/fields.yml
x-pack/filebeat/input/awscloudwatch/_meta/terraform/README.md
x-pack/filebeat/input/awscloudwatch/config.go
x-pack/filebeat/input/awscloudwatch/input.go
x-pack/filebeat/module/awsfargate/log/config/aws-cloudwatch.yml
x-pack/filebeat/module/awsfargate/log/manifest.yml
```

Are they supposed to be updated too?

dharada commented:
@rdner 

[1]
I got the following error when run `make update`.  Any idea how to fix ?

```
make update
Generating NOTICE
go mod tidy
go mod download
go list -m -json all | go run go.elastic.co/go-licence-detector \
	-includeIndirect \
	-rules dev-tools/notice/rules.json \
	-overrides dev-tools/notice/overrides.json \
	-noticeTemplate dev-tools/notice/NOTICE.txt.tmpl \
	-noticeOut NOTICE.txt \
	-depsOut ""
2022/05/02 17:14:08 Failed to detect licences: failed to detect licence type of github.com/ajstarks/svgo from /home/daisuke/.gvm/pkgsets/go1.16.3/global/pkg/mod/github.com/ajstarks/svgo@v0.0.0-20180226025133-644b8db467af/LICENSE: failed to detect licence type of /home/daisuke/.gvm/pkgsets/go1.16.3/global/pkg/mod/github.com/ajstarks/svgo@v0.0.0-20180226025133-644b8db467af/LICENSE
exit status 1
make: *** [Makefile:175: notice] Error 1
```

This PR is just for fixing a typo on [this](https://www.elastic.co/guide/en/beats/filebeat/current/exported-fields-aws-cloudwatch.html) page.  I'd prefer to defer to _docs_ team for remaining tasks if possible. 

see - https://github.com/elastic/beats/blob/d6aeef5a5ac51b4c1f114d34342f7fb3a8ae1289/x-pack/filebeat/input/awscloudwatch/processor.go#L64


[2]
>Are they supposed to be updated too?

Could you check with beats and docs team ?  (I believe Go and yml files are not supposed to be updated at least, but not sure..)

rdner commented:
@dharada

> 2022/05/02 17:14:08 Failed to detect licences: failed to detect licence type of github.com/ajstarks/svgo from /home/daisuke/.gvm/pkgsets/go1.16.3/global/pkg/mod/github.com/ajstarks/svgo@v0.0.0-20180226025133-644b8db467af/LICENSE: failed to detect licence type of /home/daisuke/.gvm/pkgsets/go1.16.3/global/pkg/mod/github.com/ajstarks/svgo@v0.0.0-20180226025133-644b8db467af/LICENSE
exit status 1

This looks like you have a change in your `go.mod` file locally with an added dependency, please make sure you don't have pending changes in your repository.

@elastic/obs-docs could you please confirm that this is the only place for this change?

dharada commented:
@rdner thanks, but it looks `go.mod` is updated by running `make update`..  (please let me know if i miss something what you mean.)

```
$ git checkout .
Updated 2 paths from the index

$ git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean

$ git diff

$ make update
Generating NOTICE
go mod tidy
go mod download
go list -m -json all | go run go.elastic.co/go-licence-detector \
	-includeIndirect \
	-rules dev-tools/notice/rules.json \
	-overrides dev-tools/notice/overrides.json \
	-noticeTemplate dev-tools/notice/NOTICE.txt.tmpl \
	-noticeOut NOTICE.txt \
	-depsOut ""
2022/05/02 17:41:34 Failed to detect licences: failed to detect licence type of github.com/ajstarks/svgo from /home/daisuke/.gvm/pkgsets/go1.16.3/global/pkg/mod/github.com/ajstarks/svgo@v0.0.0-20180226025133-644b8db467af/LICENSE: failed to detect licence type of /home/daisuke/.gvm/pkgsets/go1.16.3/global/pkg/mod/github.com/ajstarks/svgo@v0.0.0-20180226025133-644b8db467af/LICENSE
exit status 1
make: *** [Makefile:175: notice] Error 1

$ git status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   go.mod
	modified:   go.sum

no changes added to commit (use "git add" and/or "git commit -a")
```

rdner commented:
@dharada if it's updated by `make update` this means you have some pending code changes that introduce this new import. Make sure there are no uncommitted files before running `make update` on your branch.

dharada commented:
@rdner thanks again. Your response is so quick. :pray:  

> @dharada if it's updated by `make update` this means you have some pending code changes that introduce this new import. Make sure there are no uncommitted files before running `make update` on your branch.

I'm sure there are no uncommitted files before running `make update` on the `main` branch.  (I executed git reset --hard command and check with _status_ and _diff_ command)

````
$ git reset --hard
HEAD is now at 5129813e9c to fix typo

$ git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean

$  git diff
$  make update     
Generating NOTICE
go mod tidy 
go mod download
go list -m -json all | go run go.elastic.co/go-licence-detector \
	-includeIndirect \
	-rules dev-tools/notice/rules.json \
	-overrides dev-tools/notice/overrides.json \
	-noticeTemplate dev-tools/notice/NOTICE.txt.tmpl \
	-noticeOut NOTICE.txt \
	-depsOut ""
2022/05/02 20:37:00 Failed to detect licences: failed to detect licence type of github.com/ajstarks/svgo from /home/daisuke/.gvm/pkgsets/go1.16.3/global/pkg/mod/github.com/ajstarks/svgo@v0.0.0-20180226025133-644b8db467af/LICENSE: failed to detect licence type of /home/daisuke/.gvm/pkgsets/go1.16.3/global/pkg/mod/github.com/ajstarks/svgo@v0.0.0-20180226025133-644b8db467af/LICENSE
exit status 1
make: *** [Makefile:175: notice] Error 1
```

dedemorton commented:
Let's see if the build passes now then we can make the other changes that are required.

dedemorton commented:
I think this is a new failure caused by another PR that was merged and reverted.  I'll merge in main and hope it fixes the problem.

dharada commented:
(Just a note) I could succeed to run `make update` when I edit the _fields.yml_ file only. Thank you !  https://github.com/elastic/beats/pull/31476#issuecomment-1115180088

now I'm feeling difficulties to make PR for docs.  :laughing: 

<summary> make update outputs
  <details>

```
make update
Generating NOTICE
go mod tidy
go mod download
go list -m -json all | go run go.elastic.co/go-licence-detector \
	-includeIndirect \
	-rules dev-tools/notice/rules.json \
	-overrides dev-tools/notice/overrides.json \
	-noticeTemplate dev-tools/notice/NOTICE.txt.tmpl \
	-noticeOut NOTICE.txt \
	-depsOut ""
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/libbeat'
Installing mage v1.13.0.
go get: installing executables with 'go get' in module mode is deprecated.
	To adjust and download dependencies of the current module, use 'go get -d'.
	To install using requirements of the current module, use 'go install'.
	To install ignoring the current module, use 'go install' with a version,
	like 'go install example.com/cmd@latest'.
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
/home/daisuke/.magefile cleaned
Generated fields.yml for libbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/libbeat/fields.yml
>> Building libbeat.yml for linux/amd64
>> Building libbeat.reference.yml for linux/amd64
Updating generated files for libbeat
if [ -d /mnt/usbstick894/github/dharada/beats-for-pullrequest/libbeat/_meta/kibana ]; then \
	cp -pr /mnt/usbstick894/github/dharada/beats-for-pullrequest/libbeat/_meta/kibana/* /mnt/usbstick894/github/dharada/beats-for-pullrequest/libbeat/_meta/kibana.generated ; \
fi
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/libbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/auditbeat'
mage update
Generated fields.yml for auditbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/auditbeat/fields.yml
Generated fields.yml for auditbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/auditbeat/fields.yml
>> Building auditbeat.yml for linux/amd64
>> Building auditbeat.reference.yml for linux/amd64
>> Building auditbeat.docker.yml for linux/amd64
Generated fields.yml for auditbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/auditbeat/build/fields/fields.all.yml
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/auditbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/filebeat'
mage update
Generated fields.yml for filebeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/filebeat/fields.yml
Generated fields.yml for filebeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/filebeat/fields.yml
>> Building filebeat.yml for linux/amd64
>> Building filebeat.reference.yml for linux/amd64
>> Building filebeat.docker.yml for linux/amd64
Generated fields.yml for filebeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/filebeat/build/fields/fields.all.yml
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/filebeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/heartbeat'
Generated fields.yml for heartbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/heartbeat/fields.yml
cp: cannot stat 'monitors/active/*/_meta/kibana/*': No such file or directory
make[1]: [Makefile:18: kibana] Error 1 (ignored)
>> Building heartbeat.yml for linux/amd64
>> Building heartbeat.reference.yml for linux/amd64
>> Building heartbeat.docker.yml for linux/amd64
Updating generated files for heartbeat
mkdir -p include
go run  ../dev-tools/cmd/asset/asset.go -license ASL2 -pkg include -in fields.yml -out include/fields.go heartbeat
if [ -d /mnt/usbstick894/github/dharada/beats-for-pullrequest/heartbeat/_meta/kibana ]; then \
	cp -pr /mnt/usbstick894/github/dharada/beats-for-pullrequest/heartbeat/_meta/kibana/* /mnt/usbstick894/github/dharada/beats-for-pullrequest/heartbeat/_meta/kibana.generated ; \
fi
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/heartbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/metricbeat'
mage update
Generated fields.yml for metricbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/metricbeat/fields.yml
Generated fields.yml for metricbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/metricbeat/fields.yml
>> Building metricbeat.yml for linux/amd64
>> Building metricbeat.reference.yml for linux/amd64
>> Building metricbeat.docker.yml for linux/amd64
Generated fields.yml for metricbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/metricbeat/build/fields/fields.all.yml
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/metricbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/packetbeat'
mage update
No fields files for module applayer
No fields files for module tcp
No fields files for module udp
Generated fields.yml for packetbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/packetbeat/fields.yml
Generated fields.yml for packetbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/packetbeat/fields.yml
>> Building packetbeat.yml for linux/amd64
>> Building packetbeat.reference.yml for linux/amd64
>> Building packetbeat.docker.yml for linux/amd64
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/packetbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/winlogbeat'
mage update
>> update:moduleDocs: Collecting module documentation for powershell, security, sysmon.
>> Building winlogbeat.yml for windows/amd64
>> Building winlogbeat.reference.yml for windows/amd64
Generated fields.yml for winlogbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/winlogbeat/build/fields/fields.all.yml
Generated fields.yml for winlogbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/winlogbeat/build/fields/fields.common.yml
Generated fields.yml for winlogbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/winlogbeat/build/fields/fields.yml
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/winlogbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/functionbeat'
mage update
>> Building functionbeat.yml for linux/amd64
>> Building functionbeat.reference.yml for linux/amd64
Generated fields.yml for functionbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/functionbeat/fields.yml
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/functionbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/osquerybeat'
mage update
>> Building osquerybeat.yml for linux/amd64
>> Building osquerybeat.reference.yml for linux/amd64
Generated fields.yml for osquerybeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/osquerybeat/fields.yml
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/osquerybeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/auditbeat'
mage update
Generated fields.yml for auditbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/auditbeat/fields.yml
>> Building auditbeat.yml for linux/amd64
>> Building auditbeat.reference.yml for linux/amd64
>> Building auditbeat.docker.yml for linux/amd64
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/auditbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/dockerlogbeat'
Makefile:17: warning: overriding recipe for target 'unit-tests'
../..//dev-tools/make/mage.mk:47: warning: ignoring old recipe for target 'unit-tests'
mage update
>> update: There is no Update for The Elastic Log Plugin
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/dockerlogbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/filebeat'
mage update
No fields files for module azureeventhub
No fields files for module cloudfoundry
No fields files for module default-inputs
No fields files for module gcppubsub
No fields files for module http_endpoint
No fields files for module httpjson
No fields files for module o365audit
No fields files for module add_nomad_metadata
Generated fields.yml for filebeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/filebeat/fields.yml
>> Building filebeat.yml for linux/amd64
>> Building filebeat.reference.yml for linux/amd64
>> Building filebeat.docker.yml for linux/amd64
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/filebeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/heartbeat'
mage update
Generated fields.yml for heartbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/heartbeat/fields.yml
>> Building heartbeat.yml for linux/amd64
>> Building heartbeat.reference.yml for linux/amd64
>> Building heartbeat.docker.yml for linux/amd64
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/heartbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/metricbeat'
mage update
Generated fields.yml for metricbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/metricbeat/fields.yml
>> Building metricbeat.yml for linux/amd64
>> Building metricbeat.reference.yml for linux/amd64
>> Building metricbeat.docker.yml for linux/amd64
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/metricbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/winlogbeat'
mage update
>> update:moduleDocs: Collecting module documentation for powershell, security, sysmon.
>> Building winlogbeat.yml for windows/amd64
>> Building winlogbeat.reference.yml for windows/amd64
No fields files for module routing
Generated fields.yml for winlogbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/winlogbeat/build/fields/fields.all.yml
Generated fields.yml for winlogbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/winlogbeat/build/fields/fields.yml
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/winlogbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/packetbeat'
mage update
Generated fields.yml for packetbeat to /mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/packetbeat/fields.yml
>> Building packetbeat.yml for linux/amd64
>> Building packetbeat.reference.yml for linux/amd64
>> Building packetbeat.docker.yml for linux/amd64
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/packetbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/libbeat'
Makefile:12: warning: overriding recipe for target 'update'
../../libbeat/scripts/Makefile:353: warning: ignoring old recipe for target 'update'
Makefile:12: warning: overriding recipe for target 'fields'
../../libbeat/scripts/Makefile:344: warning: ignoring old recipe for target 'fields'
Makefile:12: warning: overriding recipe for target 'config'
../../libbeat/scripts/Makefile:348: warning: ignoring old recipe for target 'config'
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/x-pack/libbeat'
make[1]: Entering directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/deploy/kubernetes'
Generating filebeat-kubernetes.yaml
Generating metricbeat-kubernetes.yaml
Generating auditbeat-kubernetes.yaml
Generating heartbeat-kubernetes.yaml
make[1]: Leaving directory '/mnt/usbstick894/github/dharada/beats-for-pullrequest/deploy/kubernetes'
 
```

</details>
</summary>

dedemorton commented:
Before we merge this PR, I want to verify that this change ( `aws-cloudwatch` to `awscloudwatch`) is only required where we show exported field names. 

@kvch Can you take a look? Also let me know if we should change [the key value](https://github.com/elastic/beats/pull/31476/files#diff-3a43284a1f36db9895a39c4e31a2edc079fed754bed587a9fd033eb0078bc74eR1), too. I didn't want to change the key because it affects the URL, but I can if you want me to.
